### PR TITLE
Gushers v3.0 (I'm going to go postal if I keep coding gushers edition)

### PR DESCRIPTION
**Pull request by iskyp**
[proof that it works and i'm not going to break everything](https://imgur.com/a/gKqxj)
**CHANGELOG**

- Fixed Massive Tropical Brain Hemorrhage Gushers not appearing in hacked vending machines

- Added Hellacious Blue Phlegm Aneurysm Gushers.
Chemistry recipe is 1 part Tricordrazine, 1 part Massive Tropical Brain Hemorrhage Gusher Juice. 
Heals 3 of every type of common damage per tick.

- Added Bodacious Black Liquid Sorrow Gushers.
Chemistry Recipe is 1 part Hellacious Blue Phlegm Aneurysm Gusher Juice, 1 part oil. 
Deals 3 toxin damage per tick.

- Added Jammin Sour Diabetic Coma Gushers.
Obtainable by hacking Getmore Chocolate Vendors. No special properties.

- Added Treachous Teal Tangerine Gushers.
Obtainable as a Chef or Chemist via Syndicate Uplink. Costs 1 TC for a box of it.
Deals 5 toxin damage per tick.

- Added Wild Cherry Apeshit Apocalypse Gushers.
Chemistry recipe is 1 part Massive Tropical Brain Hemorrhage Gusher Juice, 1 part Cherry Jelly.
Increases speed to GOTTAGOREALLYFAST, and reduces stuns by quite a bit, but deals 3u of toxin damage per tick.
Overdosing it (10u) will make you drop on the floor and be stunned until it leaves your system.

- Added Carnivorous Citrus Piss Gushers.
Chemistry Recipe is 1 part Jammin Sour Diabetic Coma Gusher Juice, 1 part Lemon Lime.
While it's in your system, you will constantly puke.

- Added Xtreme Kiwi Xplosion Gushers.
Chemistry recipe is 2 parts Bodacious Black Liquid Sorrow Gusher Juice, 2 parts Carnivorous Citrus Piss Gusher Juice, 1 part Jammin Sour Diabetic Coma Gusher Juice.
As soon as you ingest it, you will obtain instant appendicitis. Fun.
Oh yeah, and as long as it's in your system, you take 4u of toxin damage per tick.

- Added Schizophrenic Strawberry Slam Gushers.
Chemistry Recipe is 1 part Massive Tropical Brain Hemorrhage Gusher Juice, 1 part Space Drugs.
Basically, you get high as fuck. That's it. Hallucinate, get the high filter on your screen, look at the colors of the wind, etc.

- Added "C O N S U M E" crate to hacked cargo consoles.
The crate costs 10000 credits, and contains:
3x tropical fruit gushers
3x sugary fruit gushers
3x hellacious fruit gushers
3x black fruit gushers
3x cherry fruit gushers
3x citrus fruit gushers
3x kiwi fruit gushers
3x strawberry fruit gushers

### CODE REVIEW

**Comment by Putnam3145** [comment]
if this doesn't work i'll break you
